driver/vhci/usbip_vhci.vcxproj

```diff
@@ -77,13 +77,15 @@
     <DriverTargetPlatform>Desktop</DriverTargetPlatform>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
-    <TargetVersion>Windows10</TargetVersion>
+    <TargetVersion>
+    </TargetVersion>
     <UseDebugLibraries>true</UseDebugLibraries>
     <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
     <ConfigurationType>Driver</ConfigurationType>
     <DriverType>KMDF</DriverType>
     <DriverTargetPlatform>Desktop</DriverTargetPlatform>
     <SpectreMitigation>false</SpectreMitigation>
+    <_NT_TARGET_VERSION>0xA000007</_NT_TARGET_VERSION>

```

Comment:
This property does not seem to be supported by VS 2017. Without this change, VS 2017 is also OK. 
I would drop this change if there's no special reason.

Reply:
This force target to Windows 10. It's not necessary. Driver should work fine.
But I suggest to test driver well before we remove this setting. (I can do it in our server farm)